### PR TITLE
Fix pylint warnings for indentation; multiple and useless imports

### PR DESCRIPTION
**Pull request by pgombar**
## Description

Fixing the following pylint warnings:

- `bad-indentation<W0311>`
- `multiple-imports<C0410>`
- `unnecessary-semicolon<W0301>`
- `useless-import-alias<C0414>`

---

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and Travis.CI is passing.

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/WALinuxAgent/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by kevinclark19a — APPROVED** [review]

File: azurelinuxagent/common/osutil/freebsd.py [hunk 1]
```diff
@@ -22,13 +22,14 @@
 import azurelinuxagent.common.utils.fileutil as fileutil
 import azurelinuxagent.common.utils.shellutil as shellutil
 import azurelinuxagent.common.utils.textutil as textutil
-from azurelinuxagent.common.utils.networkutil import RouteEntry # pylint: disable=W0611
+from azurelinuxagent.common.utils.networkutil import RouteEntry  # pylint: disable=W0611
```
Comment (larohra) [inline azurelinuxagent/common/osutil/freebsd.py:25]: Somehow I dont get these errors in pycharm (the extra space required ones)

**Review by larohra — APPROVED** [review]

File: azurelinuxagent/common/osutil/freebsd.py (hunk 1 above)
Comment (pgombar) [reply azurelinuxagent/common/osutil/freebsd.py:25]: Interesting, I didn't modify any of the default settings around whitespace... I'm using the 2020.2 version

Comment (narrieta) [reply azurelinuxagent/common/osutil/freebsd.py:25]: I get those as well

Comment (narrieta) [reply azurelinuxagent/common/osutil/freebsd.py:25]: from PEP 8
    
    An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.

**Review by narrieta — APPROVED** [review]

File: azurelinuxagent/common/osutil/freebsd.py (hunk 1 above)
Comment (larohra) [reply azurelinuxagent/common/osutil/freebsd.py:25]: Woah nice! I guess I need to fix my pycharm to actually complain about these :D

**Pull request merged**